RetailCoder.VBE/UI/RubberduckUI.de.resx

```diff
@@ -1120,4 +1120,61 @@ Warnung: Alle eigenen Einstellungen gehen dabei verloren.</value>
   <data name="Inspections_MultilineParameter" xml:space="preserve">
     <value>Variablen in einer Zeile angeben</value>
   </data>
-</root>
+  <data name="WriteOnlyProperty_" xml:space="preserve">
+    <value>Die Eigenschaft {0} hat kein 'Get'.</value>
+  </data>
+  <data name="RegexSearchReplaceScope_CurrentFile" xml:space="preserve">
+    <value>Aktuelle Datei</value>
+  </data>
+  <data name="RegexSearchReplaceScope_CurrentBlock" xml:space="preserve">
+    <value>Aktueller Block</value>
+  </data>
+  <data name="RegexSearchReplaceScope_AllOpenProjects" xml:space="preserve">
+    <value>Alle geöffneten Projekte</value>
+  </data>
+  <data name="RegexSearchReplaceScope_CurrentProject" xml:space="preserve">
+    <value>Aktuelles Projekt</value>
+  </data>
+  <data name="RegexSearchReplaceScope_AllOpenedFiles" xml:space="preserve">
+    <value>Alle geöffneten Dateien</value>
+  </data>
+  <data name="RegexSearchReplace_ReplaceButtonLabel" xml:space="preserve">
+    <value>Ersetzen</value>
+  </data>
+  <data name="RegexSearchReplace_ReplaceAllButtonLabel" xml:space="preserve">
+    <value>Alles Ersetzen</value>
+  </data>
+  <data name="RegexSearchReplace_FindButtonLabel" xml:space="preserve">
+    <value>Suche</value>
+  </data>
+  <data name="RegexSearchReplace_Caption" xml:space="preserve">
+    <value>Regex Suchen &amp; Ersetzen</value>
+  </data>
+  <data name="QuickFix_ThisProject" xml:space="preserve">
+    <value>Alle Instanzen des Projektes</value>
```

Comment:
@Vogel612 isn't there a, uh, shorter way to say this? "Fix throughout project" or something? Not a biggie since the label is actually in a `WrapPanel`, but I find the wording pretty verbose for a button ;-) (says the Canadian guy that can barely order ein bier bitte)